src/chrome/content/rules/developer.valvesoftware.com.xml

```diff
@@ -0,0 +1,16 @@
+<!--
+	Other Valve rulesets:
+		- Dota_2.com.xml
+		- Steam.xml
+		- Steam_Community.com.xml
+		- Steam_static.com.xml
+		- Steamgames.com.xml
+-->
+
+<ruleset name="Valve Developer Community">
+	<target host="developer.valvesoftware.com" />
```

Comment:
See also:
```
valvesoftware.com
www.valvesoftware.com
blogs.valvesoftware.com
list.valvesoftware.com
sources.valvesoftware.com
store.valvesoftware.com

Reply:
Which of them work?

Reply:
`^`, `www` and `list`. You can easily check it in your browser or by command line.

Reply:
`blogs` is refused, `sources` and `store` have a mismatching cert.